jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -176,18 +255,83 @@ static void afterClinitInitialization() {
 		/*[ELSE]*/
 		StringCoding.encode(new char[1], 0, 1);
 		/*[ENDIF]*/
-		/*[IF (Sidecar18-SE-OpenJ9|Sidecar19-SE)&!(PLATFORM-mz31|PLATFORM-mz64)]*/
-		setErr(new PrintStream(new BufferedOutputStream(new FileOutputStream(FileDescriptor.err)), true));
-		setOut(new PrintStream(new BufferedOutputStream(new FileOutputStream(FileDescriptor.out)), true));
+
+		/*[IF Sidecar18-SE-OpenJ9 | JAVA_SPEC_VERSION >= 11]*/
+		Properties props = internalGetProperties();
+		Charset consoleCharset = Charset.defaultCharset();
+		/*[IF PLATFORM-mz31|PLATFORM-mz64]*/
+		String systemEncoding = props.getProperty("ibm.system.encoding");
```

Comment:
This variable could be inlined like other property accesses below.
Also, the `//$NON-NLS-1$` comment is missing.